microceph/ceph/osd.go

```diff
@@ -954,27 +954,55 @@ func outDownOSD(osd int64) error {
 	return nil
 }
 
-func safetyCheckStop(osd int64) error {
+func osdNooutFlag(set bool) error {
```

Comment:
**nitpick:** 

Change function name to `setOSDNoOut`?
End with *Flag* seems like an object.

Reply:
Can be resolved.